.github/workflows/public-ci.yml

```diff
@@ -34,10 +34,17 @@ jobs:
     needs: [test]
     if: ${{ always() }}
     steps:
+      - name: Clean up github vars for posting to Slack
```

Comment:
Can we do this in a different PR? Let's just get the tests going in this PR with a simple commit message that doesn't use quotes and then you can fix the slack message in another PR?